src/app/pages/viewing/messages/messages.component.css

```
@@ -20,71 +9,122 @@
   font-size: 1em;
 }
 
+/* Message Card */
+.card {
+  display: inline-block;
+}
+
+.message-card {
+  position: relative;
+  background-color: var(--light-grey);
+  width: 100%;
+  word-break: break-word;
+  max-height: 30em;
+  overflow: hidden;
+  cursor: pointer;
+}
+
 .table td {
   overflow-wrap: break-word;
 }
 
+/* Message Carousel Page */
 .page {
   background-color: var(--accent);
   box-shadow: 0 0.15em 0.25em 0.05em rgba(0, 0, 0, 0.175);
   color: white;
   height: 70vh;
-  padding: 3em 6em 3em 6em;
-  display: flex;
+  padding: 1.5em;
   overflow-y: auto;
 }
 
-.card-header-text {
+.carousel-controls {
+  display: flex;
+  justify-content: space-between;
+  padding: 1em 2em 1em 2em;
```

```suggestion
  padding: 1em 2em;
```
Redundant because if you only use two values then CSS assumes the first one is for top/bottom and the second is for left-right.